### ISSUE TITLE
Duplicate Content-Length header on less equal then Android 5.0.

### ISSUE THREAD

**Issue by yashims** [ktorio/ktor#1151]
### Ktor Version
1.2.0

### Ktor Engine Used(client or server and name)
client

### JVM Version, Operating System and Relevant Context
Android Studio 3.4.1
Build #AI-183.6156.11.34.5522156, built on May 2, 2019
JRE: 1.8.0_152-release-1343-b01 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.14.5

#### Steps to reproduce:
1. Write and run with Android 5.0(Lolipop) or Android 4.4(KitKat)
```kotlin:sample
runBlocking {
    val url = Url("http://localhost/api/example")
    val call = HttpClient().call(url) {
        method = HttpMethod.Post
        body = FormDataContent(Parameters.build {
            append("device_id", "fiz")
        })
    }
    Log.d("REQ-HEAD ${call.request.headers}")
}
```

#### Results:
logcat
```logs:logcat
05-24 19:46:12.686 9704-9704/? D/dev.yashims: REQ-HEAD Headers [Accept-Charset=[UTF-8], Accept=[*/*]]
```

tcpdump
```logs:tcpdump
19:46:13.498661 IP 127.0.0.1.65530 > 127.0.0.1.http: Flags [P.], seq 0:299, ack 1, win 2058, options [nop,nop,TS val 887669238 ecr 3990586885], length 299: HTTP: POST /api/example HTTP/1.1
...w.HHe..'...E.._..@.@.ms
..".c. ...P..;..[.....
)I.....
4.......POST /api/example HTTP/1.1
Accept-Charset: UTF-8
Accept: */*
User-Agent: Ktor client
Content-Type: application/x-www-form-urlencoded; charset=UTF-8
Content-Length: 13
Host: localhost
Connection: Keep-Alive
Accept-Encoding: gzip
Content-Length: 13

device_id=fiz
```

And Nginx was respond 400 Bad Request.

----

* This issue happend with having post payload and on less equal then Android 5.0.
* On the other case, Not duplication Content-Length header, and respond 200 OK.
* And, same results when use other payload builder.
* I'm not yet try to use http engine other then default.

Regards.

**Comment by yashims** [comment]
I used OkHttp engine for fix this problem in the case of Android 5.0.
Or less than it, I used bad knowhow for fix this problem as first aid.

I taken as below:
1. Using OkHttp engine. (for Android 5.0) 
2. Force specifing lower OkHttp version (v3.12.x). (for Android 4.4)

```gradle
dependencies {	
    configurations.all {
        resolutionStrategy.force "com.squareup.okhttp3:okhttp:3.12.3"
    }
}
```

Note:
* **You will make not receive support if you use this method.**
* **Please do at your own risk.**

Thank you.

**Comment by yashims** [comment]
This issue duplicate with #1060.
And Fixed in Ktor v1.2.1
